web/src/flow/stages/identification/IdentificationStage.ts

```diff
@@ -274,6 +278,18 @@ export class IdentificationStage extends BaseStage<
                   `
                 : nothing}
             ${this.renderNonFieldErrors()}
+            ${this.challenge.captchaStage
+                ? html`
+                      <input name="token" type="hidden" .value="${this.token}" />
+                      <ak-stage-captcha
+                          embedded
+                          .challenge=${this.challenge.captchaStage}
+                          .onTokenChange=${(token: string) => {
+                              this.token = token;
+                          }}
```

Comment:
**This is not a change request. It is here to clarify a comment above.**

The purpose on `onTokenChange` here is not to call `submit`, it's to send a function down into `ak-stage-captcha` that will *call back*, literally, with a value to set on the client `ak-stage-identification` object.  

If this is the only place where you use `ak-stage-captcha`, you might not think it matters; the default function will always be overwritten, right?  But it's _confusing_, and six months from now some poor fellow may have to make more changes, and reading that they'll have no idea why the default function has that submit clause in it.  And there's a very good chance that poor fellow will be _you_, and you'll have done, learned, and forgotten so many things in that time you won't remember at all why it is the way it is.